source/components/Tooltip.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SkinnableComponent from './SkinnableComponent';
+
+export default class Tooltip extends SkinnableComponent {
+
+  static propTypes = Object.assign({}, SkinnableComponent.propTypes, {
+    isAligningRight: PropTypes.bool,
+    isOpeningUpwards: PropTypes.bool,
+    isBounded: PropTypes.bool,
+    tip: PropTypes.oneOfType([
```

Comment:
Please import and use `StringOrElement` from `source/utils/props.js` instead 😉

Reply:
🤙